server/performance/metrics.go

```diff
@@ -15,23 +15,27 @@ import (
 )
 
 const (
-	metricsNamespace        = "mattermost_plugin_calls"
-	metricsSubSystemWS      = "websocket"
-	metricsSubSystemCluster = "cluster"
-	metricsSubSystemStore   = "store"
+	metricsNamespace             = "mattermost_plugin_calls"
+	metricsSubSystemWS           = "websocket"
+	metricsSubSystemCluster      = "cluster"
+	metricsSubSystemStore        = "store"
+	metricsSubSystemLiveCaptions = "live_captions"
```

Comment:
I feel the subsystem should be jobs here so that it can be easily extended to more stuff in the future. Then there could be a job type label added to identify what we are talking about or a simple prefix to the metric name.

Reply:
Good idea, done.